src/components/templates/Playa/AvatarLayer.tsx

```diff
@@ -87,7 +87,8 @@ const AvatarLayer: React.FunctionComponent<PropsType> = ({
   const myAvatarRef = useRef<HTMLDivElement>(null);
 
   const { openUserProfileModal } = useProfileModalControls();
-  const { recentVenueUsers } = useRecentVenueUsers();
+  // @debt we don't have access to venueName here (but is legacy Playa template code to be removed so we don't mind), this is basically a noop
```

Comment:
What is the state of that clean up PR that removes most if not all of the legacy code? Can we merge that to avoid patching things like this one because of typescript?

Reply:
It's stale and way down on our list of priorities right now. There's no issues with doing things like this until we're able to have the breathing room to land it.